### DISCUSSION TITLE
Beginners guide to using any of this?

### DISCUSSION THREAD

**Discussion by jlive24** [cwendt94/espn-api#135] [General] [answered]
Could someone provide any sort of walk through for someone who knows next to nothing about any of this? I've downloaded Python and VSCode, and downloaded the ESPN-Fantasy-Football-API-master zip file. What do I need to do in order have use this with my league?

**Comment by cwendt94** [comment, marked as answer]
Hi @jlive24 it really depends on how you want to use it, but a good starting point for understanding the package is going to the [Wiki](https://github.com/cwendt94/espn-api/wiki). You can find your ESPN league id from the url when going to ESPN fantasy football website. For example https://fantasy.espn.com/football/team?leagueId=1234&teamId=7
